### ISSUE TITLE
is this repo dead?

### ISSUE THREAD

**Issue by janstuemmel** [ewhitley/CDAKit#2]
theres a useful PR...

**Comment by jrgerace** [comment]
I brought this repo up to support latest Swift 3, but I do not think the author is working on this anymore. (As the PR I did has not been responded to).

**Comment by janstuemmel** [comment]
maybe u can register your fork to cocoapods if youre willing to maintain that library...

would be nice, i think this library will get popular in the future...

**Comment by ewhitley** [comment]
Sorry traveling until Feb. Glad to publish as soon as I have more than iPhone on me. Tyvm for putting that work in.

**Comment by ewhitley** [comment]
While I have internet access... sore spot for me are my obviously rushed and bad test cases. They are a mess. If there are ones you'd find helpful, please do push those. Glad to hear you're using the library. Done after work because I just wanted to see one out there. Hope it's of use to you.

**Comment by janstuemmel** [comment]
ehm, yes i would, but iam not a good ios developer
... want to write a wrapper for react native

btw thanks for that lib

since youre active again i use jrgarece s in my pods

**Comment by ewhitley** [comment]
Dumb question - are people using it? If so, I'd love to how. <--- author 

Even with Apple's updated HealthKit I still don't see convenient exposure of the cda import/export functions, so I believe there is still a pupose for this library. Would love to know what others think.

**Comment by jrgerace** [comment]
The test cases are great actually but what is missing is more actual real life CDA files. The ones historically in all existing sites that I found published were all very similar, to contrast with one I pulled for myself from my own physician an Epic EHR. CDAKit successfully read it however!

**Comment by jrgerace** [comment]
It has great potential for working in HealthKit, which is why I ported it to Swift 3 so it could be used.

**Comment by janstuemmel** [comment]
iam not using it succuessfully atm (playing around with cda atm), but i want to use it for a app for my  bachelor thesis, but when iam in a bit more i will write u

**Comment by ewhitley** [comment]
@jrgerace re: real world CDA tests - couldn't agree more on the new for more test files. Finding real ones online is a bit difficult. I used my own downloaded from a few providers' EMRs, but I wasn't willing to publish those :).  I was glad to find the BCH ones - at least structurally they're good for testing. For me the larger problem has been the (lack of standards-based) coded data inside of the files. My original goal was to actually use the coded data beyond just showing the narrative form to end-users (EX: let's try using the RxNorm API, for example - read the terms from the patient meds, consolidate them, then use the API to look up drug information - same with labs in LOINC, findings in SNOMED CT, etc.). That's still ultimately my desired end-goal. I'd love to write a simple uniform light-weight wrapper around (term vocab, term value) and pull up the related data from a related term service.

**Comment by jrgerace** [comment]
@ewhitley I agree with you about the standards issue. If one could take one of the largest EHR/EMR's out there (like Epic) and maybe concentrate on their standards maybe a large coverage could be achieved maybe? RxNorm for meds, LOINC for labs, findings with SNOMED CT, or just support these here in CDAKit to start, then it might have immediate use for apps. That service if provided as a light wrapper web service would be the solution.

**Comment by antoniogermano** [comment]
I love all the work you've all put into this library. And I'd love to contribute to it, too. It's been two years, though, with no updates or even talks about this... is it ok if I fork it and try to keep it up-to-date myself?

Maybe we should team up (if you have time), gather people who are interested on this and keep it alive together... this library really is needed to interface with HealthKit's CDA, and I would love to make it more popular.
